Package.swift

```diff
@@ -80,6 +80,7 @@ let swiftPMProduct = (
         "LLBuildManifest",
         "SourceKitLSPAPI",
         "SPMLLBuild",
+        "SwiftBuildSupport",
```

Comment:
Has a use of this been accidentally introduced somewhere? We intentionally are not including SwiftBuildSupport in the SwiftPMDataModel library right now because we don't want SourceKit-LSP to pick it up as a transitive dependency

Reply:
hmm, I don't think that was intentional. I'll remove it.